tests/config/database/migrations/2017_09_05_000000_create_users_table.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateUsersTable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('users', function (Blueprint $table) {
```

Comment:
I wonder if we should rely on upstream's migration(s) here (and how).

Reply:
Yup, database testing isn't the easiest thing in the world to conceptualize and configure.
Fortunately, there are a few resource out there:
- https://phpunit.de/manual/current/en/database.html
- https://github.com/orchestral/testbench
- https://github.com/orchestral/testbench-core/tree/3.5/tests/Databases